api/callbacks.go

```diff
@@ -429,8 +442,8 @@ func cQueryExternal(ptr *C.querier_t, gasLimit C.uint64_t, usedGas *C.uint64_t,
 	bz, err := json.Marshal(res)
 	if err != nil {
 		*errOut = newUnmanagedVector([]byte(err.Error()))
-		return C.GoResult_Other
+		return C.GoError_CannotSerialize
```

Comment:
Here is the only usage of `Other` (number 4), which is a renaming to `CannotSerialize` because this is the only thing number 4 is used for.